init.coffee

```diff
@@ -13,11 +13,30 @@ exports.configure = [
     message: 'Do you need netlify hosting?'
   },
   {
+    type: 'list'
+    name: 'css'
+    message: 'What would you like to use as your CSS Preprocessor?'
+    choices: ['postcss', 'stylus']
```

Comment:
i haven't seen poststylus before, but i was just thinking that we have both the `stylus.use` and the `postcss.use` and trust the developer to use them sensibly